executor/builder.go

```diff
@@ -1213,35 +1197,46 @@ func (b *executorBuilder) buildIndexLookUpReader(v *plan.PhysicalIndexLookUpRead
 		return nil
 	}
 	is := v.IndexPlans[0].(*plan.PhysicalIndexScan)
+	indexReq.OutputOffsets = []uint32{uint32(len(is.Index.Columns))}
+	var (
+		handleCol         *expression.Column
+		tableReaderSchema *expression.Schema
```

Comment:
Can we construct table schema in physical plan builder ? We can set table schema to table plan.

Reply:
add a `TODO`